src/views/standalone/LogsView.vue

```diff
@@ -0,0 +1,178 @@
+<script lang="ts" setup>
+import type { NeComboboxOption } from '@nethserver/vue-tailwind-lib'
+import {
+  getAxiosErrorMessage,
+  NeCombobox,
+  NeInlineNotification,
+  NeTextInput,
+  NeTitle,
+  NeToggle,
+  NeTooltip
+} from '@nethserver/vue-tailwind-lib'
+import { useI18n } from 'vue-i18n'
+import { computed, onMounted, ref, watch } from 'vue'
+import { ubusCall, ValidationError } from '@/lib/standalone/ubus'
+import type { AxiosResponse } from 'axios'
+import { MessageBag } from '@/lib/validation'
+
+interface LogResponse {
+  values: string[]
+}
+
+const logLimitComboboxOption: NeComboboxOption[] = [
+  {
+    id: '100',
+    label: '100'
+  },
+  {
+    id: '200',
+    label: '200'
+  },
+  {
+    id: '500',
+    label: '500'
+  },
+  {
+    id: '1000',
+    label: '1000'
+  }
+]
+
+const { t } = useI18n()
+
+const limit = ref(logLimitComboboxOption[0].id)
+const wrapRow = ref(true)
+const search = ref('')
+const pool = ref(false)
+const poolInterval = ref<number>()
+const errorBag = ref(new MessageBag())
+
+const logRef = ref<HTMLUListElement | null>(null)
+
+const data = ref<Array<string>>()
+const error = ref<Error>()
+const loading = ref(false)
+
+const logRowStyle = computed(() => {
+  return wrapRow.value ? 'break-all' : 'whitespace-nowrap'
+})
+
+watch(data, () => {
+  setTimeout(() => {
+    if (logRef.value != null) {
+      logRef.value.scrollTop = logRef.value.scrollHeight
+    }
+  }, 50)
+})
+
+watch([limit, search], () => {
+  fetchData()
+})
+
+watch(pool, () => {
+  if (pool.value) {
+    fetchData()
+    poolInterval.value = setInterval(() => {
+      fetchData()
+    }, 5000)
```

Comment:
Maybe updating the logs every 5 seconds is a bit too slow.. Is it for performance reasons?
2-3 seconds might be ok